egs/sre16/v1/local/nnet3/xvector/prepare_feats_for_egs.sh

```diff
@@ -53,18 +65,31 @@ for n in $(seq $nj); do
   utils/create_data_link.pl $featdir/xvector_feats_${name}.$n.ark
 done
```

Comment:
Looks like the call to create_data_link.pl is invoked at line 55 and again at 65. I think you should delete the first one.

Also, I don't think you need to do the create_split_dir.pl here. I think that should be the responsibility of the caller.